src/types/note.d.ts

```
@@ -2,32 +2,45 @@ export interface NoteFormControlProps {
   }>;
   trigger?: UseFormTrigger<{
     title: string;
     content: string;
+    linkUrl?: string;
   }>;
+  setValue?: UseFormTrigger<{
+    title: string;
+    content: string;
+    linkUrl?: string;
+  }>;
+  setEmbedVisible?: (val: boolean) => void;
```

ESLint

[error] 'val' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)